reference/6/Microsoft.PowerShell.Management/Add-Content.md

```diff
@@ -237,8 +252,15 @@ Accept wildcard characters: False
 
 ### -Exclude
 
-Omits the specified items. The value of this parameter qualifies the **Path** parameter. Enter a
-path element or pattern, such as **\*.txt**. Wildcards are permitted.
+Specifies, as a string array, an item or items that this cmdlet excludes in the operation.
+The value of this parameter qualifies the **Path** parameter.
+
+Enter a path element or pattern, such as `*.txt`.
+Wildcard characters are permitted.
+
+The **Exclude** parameter is effective only when the command includes the contents of an item,
+such as `C:\Windows\*`, where the wildcard character specifies the contents of the `C:\Windows`
```

Comment:
Make sure the YAML blocks "Accept wildcard characters:" has the correct true/false value for all parameters.